### PR TITLE
Task/ui397 setup uiltraio website

### PR DESCRIPTION
**Pull request by KalynValentyn**

### CODE REVIEW

File: .devops/k8s/templates/manifest-gcp.yaml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  labels:
+    monitoring: "true"
+spec:
+  ports:
+  - protocol: TCP
+    name: http
+    port: 1337
+    targetPort: 1337
+  selector:
+    app: ${APP_NAME}-deployment
```
Comment (damienleger) [inline .devops/k8s/templates/manifest-gcp.yaml:16]: Use recommended labels please.

File: .devops/k8s/templates/manifest-gcp.yaml [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        persistentVolumeClaim:
+          claimName: ${APP_NAME}-uploads-pvc
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: ${APP_NAME}-ing
+  namespace: ${KUBE_ENV}-env
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-prod-dns # cert-manager >= v0.11.0
```
Comment (damienleger) [inline .devops/k8s/templates/manifest-gcp.yaml:81]: Delete comment please

File: README.md [hunk 3]
```diff
@@ -1,3 +1,37 @@
 # Strapi application
 
 yarn strapi start
+
+
+## DevOps
+
+### Overview
+
+Project under gitlab-ci ci/cd which mirrored repository from Github.
```
Comment (damienleger) [inline README.md:10]: Report the fixes I made here please https://github.com/ultraio/ultra-io-website-frontend/pull/3/commits/0ae52ae3a2bf9ba7c772ccf50259f0a52ccc0360

File: .gitlab-ci.yml [hunk 4]
```diff
@@ -0,0 +1,181 @@
+---
```
Comment (damienleger) [inline .gitlab-ci.yml:1]: See 
https://github.com/ultraio/ultra-io-website-frontend/pull/3/files/41a81ada7172922ed96322f71a1a850891463541#r401471881
and
https://github.com/ultraio/ultra-io-website-frontend/pull/3/files/41a81ada7172922ed96322f71a1a850891463541#r401472315

Comment (damienleger) [reply .gitlab-ci.yml:1]: actually, all the remarks on the other PR apply (mostly)

File: .devops/k8s/templates/prod-configmap.yaml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ${APP_NAME}-cm
+  namespace: prod-env
+data:
+  NODE_ENV: production
```
Comment (damienleger) [inline .devops/k8s/templates/prod-configmap.yaml:7]: Can use one file instead of two for CM

File: config/environments/production/server.json [hunk 6]
```diff
@@ -1,5 +1,5 @@
 {
```
Comment (damienleger) [inline config/environments/production/server.json:1]: Files are the same between staging & prod. Why bother having two?

File: .devops/k8s/templates/manifest-gcp.yaml [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+  name: ${APP_NAME}-svc
+  namespace: ${KUBE_ENV}-env
+  labels:
+    monitoring: "true"
+spec:
+  ports:
+  - protocol: TCP
+    name: http
+    port: 1337
+    targetPort: 1337
```
Comment (damienleger) [inline .devops/k8s/templates/manifest-gcp.yaml:14]: If it's http, better stay on 80. More speaking that 1337.

File: .devops/k8s/templates/manifest-gcp.yaml [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+                topologyKey: kubernetes.io/hostname
+      nodeSelector:
+        role: application
+      volumes:
+      - name: ${APP_NAME}-db-pvc
+        persistentVolumeClaim:
+          claimName: ${APP_NAME}-db-pvc
+      - name: ${APP_NAME}-uploads-pvc
+        persistentVolumeClaim:
+          claimName: ${APP_NAME}-uploads-pvc
```
Comment (damienleger) [inline .devops/k8s/templates/manifest-gcp.yaml:73]: What the purpose of these volumes? Applications are not supposed to be stateless?

File: .devops/k8s/templates/manifest-gcp.yaml [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+          ports:
+            - containerPort: 1337
+          volumeMounts:
+            - name: ${APP_NAME}-db-pvc
+              mountPath: /usr/src/app/.tmp
+            - name: ${APP_NAME}-uploads-pvc
+              mountPath: /usr/src/app/public/uploads
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
```
Comment (damienleger) [inline .devops/k8s/templates/manifest-gcp.yaml:58]: Better rely on requiredDuringScheduling...

File: .devops/k8s/templates/manifest-gcp.yaml (hunk 8 above)
Comment (KalynValentyn) [reply .devops/k8s/templates/manifest-gcp.yaml:73]: technically speaking:
1) for db(sqlite)
2) for uploads.
I hove pointed that we cant scale app with this method, everyone ok with this.

File: config/environments/production/server.json (hunk 6 above)
Comment (KalynValentyn) [reply config/environments/production/server.json:1]: i think remove it's wrong to remove flexibility of custom configuration

File: .devops/k8s/templates/prod-configmap.yaml (hunk 5 above)
Comment (KalynValentyn) [reply .devops/k8s/templates/prod-configmap.yaml:7]: i think it's wrong to remove flexibility of custom configuration

File: .devops/k8s/templates/manifest-gcp.yaml (hunk 9 above)
Comment (KalynValentyn) [reply .devops/k8s/templates/manifest-gcp.yaml]: removed since replica will be only 1

File: .devops/k8s/templates/manifest-gcp.yaml (hunk 8 above)
Comment (damienleger) [reply .devops/k8s/templates/manifest-gcp.yaml:64]: I think it's good point to mention in the README. So we won't be tempted to increase REPLICA in the future.

**Pull request merged**